assets/src/scss/new-identity/style.scss

```diff
@@ -1,13 +1,25 @@
 @import "../base/tokens";
+@import "../base/variables";
+@import "../base/mixins";
 
 @mixin shared-link-styles {
+  --link--height: 26px;
+  --transition--duration: 0.3s;
+
   border-bottom: 2px solid var(--gp-green-500);
-  transition: 0.3s cubic-bezier(0.5, 0, 0, 1);
+  transition: var(--transition--duration) cubic-bezier(0.5, 0, 0, 1);
+
+  &:hover {
+    box-shadow: inset 0 calc(var(--link--height) * -1) rgba($gp-green-500, .15);
+  }
 
   &:hover,
+  &:active {
+    --link--hover--text-decoration: none;
```

Comment:
I still think here it would be simpler to just remove these overrides in the Split Two Columns styles since they are useless anyway. And then here we could just use `text-decoration`.
But if you think that's too many changes you can also revert to using `!important`, I guess that's not a big deal for now and we can figure out the refactor when merging the new identity styles...

Reply:
From my point of view, for some reasons I'd prefer to use `!important` even if it isn't the best approach.

- We'll keep the current behavior up to date and nothing could be potentially broken. If something is broken then we'll blame the `new-identity` stylesheet 😆.  
- Looking at the `!important` in the future, we'll understand that that part of the code needs to be evaluated after we'll merge the new feature.

Reply:
Yeah I'm fine with that, your arguments make sense! 👍